### DISCUSSION TITLE
Gauge charts through Netdata data collection plugin

### DISCUSSION THREAD

**Discussion by rshen-tibco** [netdata/netdata#15931] [Q&A]
I'm aware that there are chart libraries for making gauge/pie charts, but I was wondering how the go.d.plugin for Pulsar managed to get them to show up as well.

<img width="485" alt="image" src="https://github.com/netdata/netdata/assets/85526583/46bca66c-91fe-499a-b770-9c2770347a57">

I'm not really seeing anything gauge- or pie-related in the netdata/go.d.plugin/modules/pulsar code. Any idea how they're getting that to show up or what another data collection plugin could add in their code to get similar charts?

**Comment by ilyam8** [comment]
Hi, @rshen-tibco. It is not possible to add custom gauges at the moment. The ones you see are hard-coded in the UI code.

**Reply by rshen-tibco** [reply]
Do you know where the code is (like which section of while file) for the Pulsar charts? I'm just curious because Pulsar is an integration rather than something that's automatically loaded like the System information charts, and they're also able to get gauge charts.
